aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -225,7 +225,7 @@ bool check_sm_version(cudaDeviceProp * dprops) {
 bool check_flash_attention_hardware_support(sdp_params const& params, bool debug) {
   // Check that the gpu is capable of running flash attention
   using sm80 = SMVersion<8, 0>;
-  using sm120 = SMVersion<12, 0>;
+  using sm121 = SMVersion<12, 1>;
```

Comment:
Same is there no way to make the version support un upperbounded and let CUDA through a different error if so?